client/connector/EventSource.cs

```diff
@@ -15,16 +15,17 @@ public class EventSource : IService
     {
         private readonly ILogger<EventSource> logger;
         private readonly string url;
-        private readonly Config config;
         private readonly HttpClient httpClient;
         private readonly IUpdateCallback callback;
-        private const int ReadTimeoutMs = 60_000;
+        private const int ReadTimeoutMs = 35_000;
```

Comment:
FF-Server sends heartbeats every 20 - 30 seconds, so 35 seconds should be enough time to detect a dead stream while not killing alive streams.